modules/core/test/v2/integration/wallet.ts

```diff
@@ -799,4 +799,26 @@ describe('V2 Wallet:', function() {
 
   });
 
+  describe('Algo non partipation key reg transaction', function() {
```

Comment:
This integration test fails for me when run locally with the following error:
```

Error: wallet not found: 602566f550a05c0006fe13a03b57141f
    at errFromResponse (src/bitgo.ts:90:20)
    at handleResponseError (src/bitgo.ts:107:11)
From previous event:
    at Request.req.then (src/bitgo.ts:722:40)
    at Request.superagent.Request.result (src/bitgo.ts:76:15)
    at Wallets.<anonymous> (src/v2/wallets.ts:688:10)
    at step (src/v2/wallets.ts:24:23)
    at Object.next (src/v2/wallets.ts:5:53)
From previous event:
    at Wallets.getWallet (src/v2/wallets.ts:691:8)
    at Context.<anonymous> (test/v2/integration/wallet.ts:805:51)
    at step (test/v2/integration/wallet.ts:35:23)
    at Object.next (test/v2/integration/wallet.ts:16:53)
    at /home/tyler/src/bjs/modules/core/test/v2/integration/wallet.ts:10:71
    at new Promise (<anonymous>)
    at __awaiter (test/v2/integration/wallet.ts:6:12)
    at Context.<anonymous> (test/v2/integration/wallet.ts:1045:20)
```

Was this tested locally?

Reply:
Yes, this has been tested locally with testnet-01 environment. The accompanying changes on platform are not yet there in test environment. Both SDK and platform were tested in testnet-01

Reply:
As discussed, have disabled this test - marked it xit and added comment to enable it back when support is available in platform in 'test' environment